src/photosui.cs

```diff
@@ -24,8 +24,8 @@ namespace PhotosUI {
 	[NoTV]
 	[iOS (8, 0)]
 	[Mac (10, 13)]
+	[MacCatalyst (15,0)]
```

Comment:
According to web doc this was in Catalyst 14+

When working on the backlog you need to check the version where this was added (since it's not new in Xcode 13)

`/Applications/Xcode_13.0.0-beta.app//Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/PhotosUI.framework/Headers/PHContentEditingController.h`
shows this

```
API_AVAILABLE_BEGIN(macos(10.11), ios(8), macCatalyst(14))
```

since it's not new to Xcode 13 (iOS15) then `[MacCatalyst (14,0)]` makes sense (unless it fails running intro on BS)

Reply:
Ah okay I see! Thank you!